From 1ecad588cfbe2b38259b4d06c7f1843efb48e142 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Tue, 22 Aug 2017 11:32:55 +0800
Subject: [PATCH 2433/5242] MLK-16242-4 video: fbdev: dcss: realize 'fb_ioctl'
 interface

commit  6da725497749d4468c13f1ec00adc01db2fa534b from
https://source.codeaurora.org/external/imx/linux-imx.git

Realize the 'fb_ioctl' hook interface with a blank
function. Non-standard ioctl commands implementations
can be added later.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/video/fbdev/mxc/imx_dcss.c |    9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/drivers/video/fbdev/mxc/imx_dcss.c b/drivers/video/fbdev/mxc/imx_dcss.c
index 7a2b61f..ee0ce7d 100644
--- a/drivers/video/fbdev/mxc/imx_dcss.c
+++ b/drivers/video/fbdev/mxc/imx_dcss.c
@@ -514,6 +514,8 @@ static int dcss_setcolreg(unsigned regno, unsigned red, unsigned green,
 static int dcss_blank(int blank, struct fb_info *fbi);
 static int dcss_pan_display(struct fb_var_screeninfo *var,
 			    struct fb_info *fbi);
+static int dcss_ioctl(struct fb_info *fbi, unsigned int cmd,
+		      unsigned long arg);
 
 static struct fb_ops dcss_ops = {
 	.owner = THIS_MODULE,
@@ -523,6 +525,7 @@ static int dcss_pan_display(struct fb_var_screeninfo *var,
 	.fb_setcolreg	= dcss_setcolreg,
 	.fb_blank	= dcss_blank,
 	.fb_pan_display	= dcss_pan_display,
+	.fb_ioctl	= dcss_ioctl,
 	.fb_fillrect	= cfb_fillrect,
 	.fb_copyarea	= cfb_copyarea,
 	.fb_imageblit	= cfb_imageblit,
@@ -2565,6 +2568,12 @@ static int dcss_pan_display(struct fb_var_screeninfo *var,
 	return 0;
 }
 
+static int dcss_ioctl(struct fb_info *fbi, unsigned int cmd,
+		      unsigned long arg)
+{
+	return 0;
+}
+
 static void ctxld_irq_clear(struct dcss_info *info)
 {
 	uint32_t irq_status;
-- 
1.7.9.5

